src/content/docs/apm/agents/php-agent/frameworks-libraries/analyze-phpunit-test-data-new-relic.mdx

```diff
@@ -12,7 +12,16 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-If you use [PHPUnit](https://phpunit.de/) to manage and run your unit tests, the New Relic PHP agent can automatically capture the test summary results and send them to [New Relic](/docs/insights/new-relic-insights/understanding-insights/new-relic-insights) as an [event](/docs/insights/new-relic-insights/understanding-insights/new-relic-insights#database) where you can query and visualize test data at a glance. This feature was released in version <DNT>**6.7.0**</DNT> of the PHP Agent and supports PHPUnit versions <DNT>**3.7**</DNT> to <DNT>**8.x**</DNT>
+If you use [PHPUnit](https://phpunit.de/) to manage and run your unit tests, the New Relic PHP agent can automatically capture the test summary results and send them to [New Relic](/docs/insights/new-relic-insights/understanding-insights/new-relic-insights) as an [event](/docs/insights/new-relic-insights/understanding-insights/new-relic-insights#database) where you can query and visualize test data at a glance. This feature supports PHPUnit versions <DNT>**8.x**</DNT> only.
+
+<Callout variant="caution">
+Even though New Relic PHP Agent does not support PHPUnit version 11+, there's a known incompatibility between PHPUnit version 11+ and New Relic PHP Agent.
+Therefore we recommend disabling the agent while running unit tests by setting `newrelic.enabled` to `false`. This setting can be passed to `phpunit` like this:
```

Comment:
```suggestion
There is a known incompatibility between PHPUnit version 11+ and the New Relic PHP Agent.
While we don't support PHPUnit version 11+, we do recommend disabling the agent to avoid memory issues while running unit tests for PHPUnit 11+ by setting `newrelic.enabled` to `false`. This setting can be passed to `phpunit` like this:
```